src/main/java/org/spburegistry/backend/entity/RoleInProject.java

```diff
@@ -8,9 +8,9 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Entity
-@Table(name="roles")
+@Table(name="roles_in_projects")
 @Data
-public class Role extends BaseEntity {
+public class RoleInProject extends BaseEntity {
```

Comment:
I would like to rename this class like "ProjectRole" or something. No need to rename sql tables.